multires-lyrics-search/pods/chunk_merger.yml

```diff
@@ -0,0 +1,10 @@
+!BaseExecutor
+with: {}
+metas:
+  name: merge
+requests:
+  on:
+    [SearchRequest, TrainRequest, IndexRequest, ControlRequest]:
```

Comment:
When do we need TrainRequest?

Reply:
It is not curetly active the `TraninRequest` I would remove it, but what is more important is to remove `ControlRequest` from here